landingzones/views.py

```diff
@@ -480,6 +520,111 @@ def form_valid(self, form):
         )
 
 
+class ZoneUpdateView(
+    LoginRequiredMixin,
+    InvestigationContextMixin,
+    ZoneModifyMixin,
+    UpdateView,
+):
+    """LandingZone update view"""
+
+    model = LandingZone
+    form_class = LandingZoneForm
+    slug_url_kwarg = 'landingzone'
+    slug_field = 'sodar_uuid'
+
+    def get_context_data(self, *args, **kwargs):
+        context = super().get_context_data(*args, **kwargs)
+        context['zone'] = LandingZone.objects.get(
```

Comment:
As we use `UpdateView`, this should not be needed. We should be able to use `object` in the template.